scripts/needs_testrun.py

```diff
@@ -178,7 +178,14 @@ def get_patterns(suite: str) -> t.Set[str]:
         suitespec = json.load(f)
 
         compos = suitespec["components"]
-        suite_patterns = set(suitespec["suites"].get(suite, []))
+        if suite not in suitespec["suites"]:
+            return set()
+
+        suite_patterns = set(suitespec["suites"][suite])
+
+        # Include patterns from include-always components
+        for patterns in (patterns for compo, patterns in compos.items() if compo.startswith("$")):
+            suite_patterns |= set(patterns)
```

Comment:
I might be missing some context - is this addition related to the riot upgrade?

Reply:
Yes. Although @emmettbutler fixed this in another PR, I've added this new "syntax" to define patterns that always trigger a job essentially